### PR TITLE
solution

### PR DESCRIPTION
**Pull request by nrjss67**

### CODE REVIEW

**Review by vsmutok — CHANGES_REQUESTED** [review]
Tests Failed!

File: taxi/urls.py [hunk 1]
```diff
@@ -16,8 +22,38 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
```
Comment (vsmutok) [inline taxi/urls.py:29]:
```suggestion
    path(
        "manufacturers/create/",
        ManufacturerCreateView.as_view(),
        name="manufacturer-create",
    ),
```
add `/` wherever needed

File: taxi/views.py [hunk 2]
```diff
@@ -34,12 +35,38 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
...
+
+class ManufacturerDeleteView(LoginRequiredMixin, generic.edit.DeleteView):
+    model = Manufacturer
+    success_url = reverse_lazy("taxi:manufacturer-list")
+
+
 class CarListView(LoginRequiredMixin, generic.ListView):
     model = Car
     paginate_by = 5
     queryset = Car.objects.all().select_related("manufacturer")
```
Comment (vsmutok) [inline taxi/views.py:60]: `all()` redundant here

File: taxi/views.py [hunk 3]
```diff
@@ -52,3 +79,15 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```
Comment (vsmutok) [inline taxi/views.py:81]: `all()` redundant here

**Review by vsmutok — CHANGES_REQUESTED** [review]

**Review by EdAlekseiev — APPROVED** [review]

**Pull request closed without merging**